### ISSUE TITLE
Change wording re:unapproved txs on switch network window

### ISSUE THREAD

**Issue by dpazdan** [MetaMask/metamask-extension#11206]
**Describe the bug**
Currently the wording is confusing:
Switching networks will cancel all pending confirmations.

Some users think their existing transactions that already broadcast will be deleted or cancelled.

**Steps to reproduce (REQUIRED)**
Steps to reproduce the behavior, libraries used with version number, and/or any setup information to easily reproduce:

1. Add a network from chainlist.org
2. see warning at the bottom of the window.
![image](https://user-images.githubusercontent.com/16330048/120014007-24a26c00-bfa7-11eb-8835-98be834cbc85.png)

**Expected behavior**
Change wording to something like:
Switching networks will cancel unapproved transactions on existing network.  To learn more click here. With a link to a better explanation on support. 
OR
Make it conditional and just show the warning when they do have txs pending approval.  And then say: Warning the existing network has transaction pending approval,  if you switch networks you will lose these.  Click here to view your unapproved transactions.

**Comment by rachelcope** [comment]
Making it conditional would be a better user experience. Error copy could be: 

You have unapproved confirmation on the current network. Switching networks will cancel these confirmations.

**Comment by AyiSoli** [comment]
The current copy is extremely confusing for users. 
This notice is shown when transactions are pending on the network and a user changes the network to another chain. 
What happens is Metamask doesnt cancel the transaction (nor can metamask cancel it unless the user cancels it), but Metamask doesnt show transaction confirmation notifications if the user changes the network. 

So the copy could instead read "switching wallets will cancel pending confirmation notifications".

**Comment by ackvf** [comment]
Much better what @AyiSoli has proposed. The problem is that _"will cancel all pending confirmations"_ is very unfortunate wording, because we were all taught _"A **transaction is confirmed** when miners include it in a block."_, so what metamask is actually referring to is a "**confirming notification**" of a confirmed transaction.

I would rephrase it even more:

> When you switch a network, you won't receive notification when a transaction is confirmed on another network. It can still be found on blockchain explorer.

**Comment by icstm** [comment]
@ackvf and @AyiSoli have provided much better wording, can this please be actioned and added to the backlog?
The current wording is closer to be being incorrect than correct, it is so confusing!

**Comment by ackvf** [comment]
I don't remember seeing that message any more. Has it been removed completely?
